hedera/settings.py

```diff
@@ -13,6 +13,8 @@
     "default": dj_database_url.config(default="postgres://localhost/hedera")
 }
 
+DATABASE_URL = os.environ.get("DATABASE_URL")
```

Comment:
@joshuagetega this shouldn't be necessary.  `dj_database_url` above will read in `$DATABASE_URL` from env and use that instead of the `default` that is provided, if it exists.

Reply:
I see, @paltman . I've removed the line and tested the update. The db connection is made successfully without the line. Thanks for the feedback. Going ahead with the merge ...